src/plugins/messageLinkEmbeds/index.tsx

```diff
@@ -46,11 +45,12 @@ const messageCache = new Map<string, {
 }>();
 
 const Embed = LazyComponent(() => findByCode(".inlineMediaEmbed"));
-const ChannelMessage = LazyComponent(() => find(m => m.type?.toString()?.includes('["message","compact","className",')));
+const AutoModEmbed = LazyComponent(() => findByCode(".withFooter]:", "childrenMessageContent:"));
+const ChannelMessage = LazyComponent(() => find(m => m.type?.toString()?.includes("renderSimpleAccessories)")));
 
 const SearchResultClasses = findByPropsLazy("message", "searchResult");
 
-let AutoModEmbed: React.ComponentType<any> = () => null;
+const SettingsStores = findByPropsLazy("MessageDisplayCompact");
```

Comment:
```suggestion
const TextAndImageSettingsStores = findByPropsLazy("MessageDisplayCompact");
```
and maybe move to webpack/commons/settingsStores.ts?